packages/appstore/src/components/containers/__tests__/listing-container.spec.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import ListingContainer from '../listing-container';
+import { StoreProvider, mockStore } from '@deriv/stores';
+
+jest.mock('Components/pre-loader/title-card-loader', () => jest.fn(() => <span>TitleCardLoader</span>));
+jest.mock('Components/currency-switcher-card', () => jest.fn(() => <span>CurrencySwitcherCard</span>));
+jest.mock('Components/containers/grid-container', () => jest.fn(({ children }) => <>{children}</>));
+
+const Title = () => <span>Test Title</span>;
+const Description = () => <span>Test Description</span>;
+
+const mock = mockStore({});
```

Comment:
@sergei-deriv Use `mockStore` in each test case to create a fresh object for that test case 🙏🏻

Reply:
@farzin-deriv I'm not sure why we need this but ok, I've change it